homeassistant/components/tellstick.py

```diff
@@ -59,12 +59,12 @@ def _discover(hass, config, component_name, found_tellcore_devices):
 def setup(hass, config):
     """Setup the Tellstick component."""
     from tellcore.constants import TELLSTICK_DIM
-    from tellcore.library import DirectCallbackDispatcher
+    from tellcore.telldus import AsyncioCallbackDispatcher
     from tellcore.telldus import TelldusCore
 
     try:
         tellcore_lib = TelldusCore(
-            callback_dispatcher=DirectCallbackDispatcher())
+            callback_dispatcher=AsyncioCallbackDispatcher(hass.loop))
```

Comment:
What does this do? Will this allow you to have all your callbacks be coroutines?